terraform/ssh/ssh.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package ssh
+
+import (
+	"io"
+	"net"
+	"os"
+	"strings"
+
+	"golang.org/x/crypto/ssh"
+	"golang.org/x/crypto/ssh/agent"
+)
+
+// Conn is a wrapper type over a ssh connection
+// that takes care of creating a channel and running
+// commands in a single method.
+type Conn struct {
```

Comment:
I think we can rename this to `Client` :)

Reply:
done.